src/migrations/1602486053913-createTables.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020-present Open Networking Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class createTables1602486053913 implements MigrationInterface {
+    name = 'createTables1602486053913'
```

Comment:
did it generate this for you? nice

Reply:
Ya, I have the command in the readme, will be apart of the readme branch. BTW, I will be removing the add modules in the readme branch

Reply:
NVM not in it haha